view/frontend/web/js/action/express-pay/create-wallet-pay-order-action.js

```diff
@@ -20,7 +20,8 @@ define(
                     quoteMaskId: window.checkoutConfig.quoteData.entity_id,
                     publicOrderId: window.checkoutConfig.bold.publicOrderId,
                     gatewayId: paymentPayload.gateway_id,
-                    shippingStrategy: paymentPayload.shipping_strategy || 'dynamic'
+                    shippingStrategy: paymentPayload.shipping_strategy || 'dynamic',
+                    shouldVault: paymentPayload.should_vault // TODO: only include this for card payments, not wallets
```

Comment:
Is there a good way we can detect whether a given wallet-pay-order is actually a wallet pay vs ACDC card fields?